### PR TITLE
[fix #39] properly include libzip

### LINKED ISSUE #39: includes lead to using GDAL 1.11
> ../src/core/qgsgml.cpp:883:9: error: use of undeclared identifier 'OGR_G_ExportToIsoWkb'; did you mean 'OGR_G_ExportToWkb'?
        OGR_G_ExportToIsoWkb( hGeom, wkbNDR, pabyBuffer );
        ^~~~~~~~~~~~~~~~~~~~
        OGR_G_ExportToWkb
/usr/local/include/ogr_api.h:102:16: note: 'OGR_G_ExportToWkb' declared here
OGRErr CPL_DLL OGR_G_ExportToWkb( OGRGeometryH, OGRwkbByteOrder, unsigned char*)

I think there is an issue with includes with latest master of QGIS.
When runnong qgis-cmake-setup.sh and build.
`/usr/local/include/ogr_api.h` points to GDAL 1.11 while GDAL 2 is correctly setup in cmake.

### PR DESCRIPTION
**Pull request by 3nids**
kudos to @m-kuhn for helping to understand this

not sure about the formula itself, I did not tested it.
but the cmake script is working as expected for me.

### CODE REVIEW

File: Formula/qgis3-dev.rb [hunk 1]
```diff
@@ -244,6 +245,7 @@ def install
     args << "-DGSL_CONFIG=#{Formula["gsl"].opt_bin}/gsl-config"
     args << "-DGSL_INCLUDE_DIR=#{Formula["gsl"].opt_include}"
     args << "-DGSL_LIBRARIES='-L#{Formula["gsl"].opt_lib} -lgsl -lgslcblas'"
+    args << "-DLIBZIP_INCLUDE_DIR=#{Formula["libzip"].opt_include}"
```
Comment (dakcarto) [inline Formula/qgis3-dev.rb]: `LIBZIP_INCLUDE_DIR` still needs specifically set even though `libzip` opt prefix has been added to `CMAKE_PREFIX_PATH`?

If so, maybe we need to update the [`FindLibZip.cmake`](https://github.com/qgis/QGIS/blob/master/cmake/FindLibZip.cmake) instead?

File: Formula/qgis3-dev.rb [hunk 2]
```diff
@@ -216,6 +216,7 @@ def install
       flex
       bison
       fcgi
+      libzip
```
Comment (dakcarto) [inline Formula/qgis3-dev.rb]: `libzip` is already listed 4 lines above.

File: Formula/qgis3-dev.rb (hunk 1 above)
Comment (3nids) [reply Formula/qgis3-dev.rb]: not sure, I applied the same fix than for the cmake script. will remove it

File: Formula/qgis3-dev.rb (hunk 2 above)
Comment (3nids) [reply Formula/qgis3-dev.rb]: oops, removing.

**Review by 3nids — COMMENTED** [review]
I reverted the formula part, keeping only the cmake script which I tested.

**Comment by dakcarto** [comment]
Thanks Denis!

**Comment by 3nids** [comment]
thank you!

**Pull request merged**